### PR TITLE
GAWB-3671: Fix quoting when populating from JSON

### PR DESCRIPTION
**Pull request by breilly2**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Review by ahaessly — APPROVED** [review]
👍

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala [hunk 1]
```diff
@@ -107,7 +169,7 @@ class MethodConfigSpec extends FreeSpec with Matchers with WebBrowserSpec with W
   private def generateInputsJson(inputs: Map[String, String]): File = {
     val file = File.createTempFile("MethodConfigSpec_", "_inputs.json")
     val writer = new PrintWriter(file)
-    val rows = inputs map { case (k, v) => s""""$k": "$v"""" }
+    val rows = inputs map { case (k, v) => s""""$k": $v""" }
```
Comment (andy7i) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala:173]: This seems odd. Did it not work before?

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala [hunk 2]
```diff
@@ -77,26 +79,86 @@ class MethodConfigSpec extends FreeSpec with Matchers with WebBrowserSpec with W
...
+                      |  }
+                      |  output {
+                      |    String out = stdout()
+                      |  }
+                      |  runtime {
+                      |    docker: "ubuntu"
+                      |  }
+                      |}
+                      |""".stripMargin)
+        api.methods.createMethod(method.creationAttributes)
```
Comment (andy7i) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala:123]: For cleanliness, can we delete the method after the test?

**Review by ansingh7115 — APPROVED** [review]
Nice! Thanks for finding and fixing this!

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala (hunk 1 above)
Comment (breilly2) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala:173]: No all values should have quotes (e.g. numbers) so quotes are now included in the values of the `variables` map when appropriate.

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala (hunk 2 above)
Comment (breilly2) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala:123]: Best we can do is redact, but I can add that.

**Comment by breilly2** [comment]
Andrew is on vacation but I addressed his feedback so I'm going ahead with my merge. Got QA approval about failed test after it passed on a manual re-run.

**Pull request merged**